go.mod

```diff
@@ -697,11 +697,7 @@ exclude google.golang.org/grpc/examples v0.0.0-20200728065043-dfc0c05b2da9
 // Replacing for an internal fork that exposes internal folders
 // Some funtionalities of the collector have been made internal and it's more difficult to build and configure pipelines in the newer versions.
 // This is a temporary solution while a new configuration design is discussed for the collector (ref: https://github.com/open-telemetry/opentelemetry-collector/issues/3482).
```

Comment:
This issue is now closed. Is there a need to update this comment with the reason for the fork?

Reply:
Good point, thank you! I meant to update this comment but didn't forgot to do it earlier.